backend/src/main/java/com/relay/controllers/UserController.java

```diff
@@ -1,34 +1,27 @@
 package com.relay.controllers;
 
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.web.bind.annotation.DeleteMapping;
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RestController;
 
-import com.relay.model.User;
-import com.relay.service.UserService;
-
 @RestController
 public class UserController {
 
-    private final UserService userService;
-
-    @Autowired
-    public UserController(UserService userService) {
-
-        this.userService = userService;
-    }
-
-    @GetMapping("/users")
-    public Iterable<User> retrieveAllRelays() {
-
-        return userService.findAll();
-    }
-
-    @DeleteMapping("/users/{id}")
-    public void deleteUser(@PathVariable int id) {
-
-        userService.deleteById(id);
-    }
+    // private final UserService userService;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)